src/tutorials/0008-introduction-to-libp2p/03.md

```diff
@@ -0,0 +1,91 @@
+---
+    title: "Enter libp2p"
+    type: "multiple-choice"
+---
+
+`libp2p` is the networking stack of IPFS, but extracted away from IPFS, becoming a first class project and a dependency of IPFS itself.
+
+<div class="flex justify-center mt4 mb4">
+    <img class="w-75-ns w-100" src="/tutorial-assets/T0008L03-libp2p-ipfs-animation.gif" alt="libp2p animation representing libp2p being extracted out of ipfs" />
+</div>
+
+This way, `libp2p` is able to grow further without being attached specifically to `IPFS`, gaining its own ecosystem and community. `IPFS` simply becomes one of the many users of `libp2p`.
+
+This way each project can solely focus on their own objectives:
+
+- `IPFS` is more focused on *__Content Addressing__*, i.e., find, fetch and authenticate any piece of content in the web. 
+- `libp2p` is more focused on *__Process Addressing__*, i.e., find, connect and authenticate any process in the network.
```

Comment:
I don't have a good enough sense of what these processes might be to understand this.

Reply:
I believe it's an abstract way of thinking about can connections be used for: file sharing, video calls, loading a website etc.

https://youtu.be/CRe_oDtfRLw?t=552

Reply:
something like "data transfer processes" might help me here